tests/integration/buildcmd/build_integ_base.py

```diff
@@ -78,6 +78,7 @@ def get_command_list(
         build_image=None,
         exclude=None,
         region=None,
+        hook_package_id=None,
         beta_features=False,
```

Comment:
can we allow accepting None, so we can test both `--beta-features` and `--no-beta-features` use cases.